src/torchmetrics/functional/text/sacre_bleu.py

```diff
@@ -116,9 +133,18 @@ class _SacreBLEUTokenizer:
         "zh": "_tokenize_zh",
         "intl": "_tokenize_international",
         "char": "_tokenize_char",
+        "ja-mecab": "_tokenize_ja_mecab",
+        "ko-mecab": "_tokenize_ko_mecab",
+        "flores101": "_tokenize_flores_101",
+        "flores200": "_tokenize_flores_200",
     }
 
-    def __init__(self, tokenize: Literal["none", "13a", "zh", "intl", "char"], lowercase: bool = False) -> None:
+    # Keep it as class variable to avoid initializing over and over again
+    sentenpiece_processors: ClassVar[Dict[str, Optional[Any]]] = {"flores101": None, "flores200": None}
```

Comment:
I've seen that this pull request is linked to the issue I opened some days ago #2054.
I think there is a typo in this class variable, it should be `sentencepiece_processors` instead of `sentenpiece_processors`, just to conform with the package name.